src/ui/style.css

```diff
@@ -119,22 +85,8 @@
   padding: 10px;
   margin: 0 0 5px;
   border-radius: 7px;
-  color: #fff;
-}
-
-.environment-entry.gaming {
-  background: #DD1B24;
-  border: 1px solid rgba(0, 0, 0, 0.2);
-}
-
-.environment-entry.software {
-  background: #1C71D8;
-  border: 1px solid rgba(0, 0, 0, 0.2);
-}
-
-.environment-entry.custom {
-  background: #9A9996;
   border: 1px solid rgba(0, 0, 0, 0.2);
+  color: black
```

Comment:
for some reason it's white by default (couldn't figure out why)

setting this to `color: black` breaks dark mode, any idea what's the best way to set this conditionally based on dark mode being preferred or not?

I don't think GTK supports `prefers-color-scheme` but I didn't try TBH

Reply:
> for some reason it's white by default (couldn't figure out why)
> 
> setting this to `color: black` breaks dark mode, any idea what's the best way to set this conditionally based on dark mode being preferred or not?
> 
> I don't think GTK supports `prefers-color-scheme` but I didn't try TBH

prefers-color-scheme is not supported :S maybe we can offer 2 different styles and then inject when user switch the dark mode

Reply:
maybe it is white from the parent

Reply:
Thanks - I'll look into it once we have a go from designers.